kv/org.go

```diff
@@ -407,12 +407,28 @@ func (s *Service) updateOrganization(ctx context.Context, tx Tx, id influxdb.ID,
 	return o, nil
 }
 
+func (s *Service) deleteOrganizationsBuckets(ctx context.Context, tx Tx, id influxdb.ID) error {
+	filter := influxdb.BucketFilter{
+		OrganizationID: &id,
+	}
+	bs, err := s.findBuckets(ctx, tx, filter)
+	if err != nil {
+		return err
+	}
+	for _, b := range bs {
+		if err := s.deleteBucket(ctx, tx, b.ID); err != nil {
+			return err
```

Comment:
If something went wrong deleting one bucket you probably want to keep going. Maybe just log the error here instead of returning it.

Reply:
good idea